lib/rspec/rails/matchers/active_job.rb

```diff
@@ -163,7 +163,27 @@ def at_match?(job)
             return job[:at].nil? if @at == :no_wait
             return false unless job[:at]
 
-            values_match?(@at, Time.at(job[:at]))
+            scheduled_at = Time.at(job[:at])
+            values_match?(@at, scheduled_at) || check_for_inprecise_value(scheduled_at)
+          end
+
+          def check_for_inprecise_value(scheduled_at)
+            return unless Time === @at && values_match?(@at.change(usec: 0), scheduled_at)
+
+            RSpec.warn_with((<<-WARNING).gsub(/^\s+\|/, '').chomp)
```

Comment:
Is there a possibility that the user want to silence this warning?

Reply:
There is, but this is a warning that is caused in a failing scenario only, if the times don't match at all you won't see it, only if the times match except in `usec` values should you see it.

I think this is niche enough to not need suppressing, as the solution to silencing it is to fix your passed in value.

Reply:
Thanks :)